pkg/stores/proxyalpha/tablelistconvert/client.go

```diff
@@ -0,0 +1,138 @@
+/*
+Package tablelistconvert provides a client that will use a table client but convert *UnstructuredList and *Unstructured objects
+returned by ByID and List to resemble those returned by non-table clients while preserving some table-related data.
```

Comment:
The table client provides some data that UI depends on that actually is unavailable otherwise. Table returns an object with metadata.fields which is important to the ui. We index these on the fields as well. These represent notable columns. These fields looks like this:
![image](https://github.com/rancher/steve/assets/19376037/ab361cda-85ec-4131-8d90-fffa22be991c)

UI then looks at the schema for the object to know what each field represents. [We are doing this in the proxy store today](https://github.com/rancher/steve/blob/master/pkg/stores/proxy/proxy_store.go#L150-L198). In order for the informer to have access to this information I build the convert client so I could pass the client into the informer.